circuit-construction/src/writer.rs

```diff
@@ -782,7 +790,7 @@ pub trait Cs<F: PrimeField> {
                                         // Do one full round on the previous value
                                         let mut acc = states[states.len() - 1]
                                             .iter()
-                                            .map(|x| x.val())
+                                            .map(crate::writer::Var::val)
```

Comment:
don't think this is a good change :o

Reply:
Do you think that way?
So, you mean better to keep original style? like `(|x| x.val())`